Content.Server/StationEvents/Components/MeteorSwarmComponent.cs

```diff
@@ -18,7 +18,7 @@ public sealed partial class MeteorSwarmComponent : Component
     public int WaveCounter;
 
     [DataField]
-    public float MeteorVelocity = 10f;
+    public float MeteorVelocity = 5f;
```

Comment:
If the point is to avoid them hitting shuttles and players as often, change the targeting for meteors from being at players to being at random tiles on the station.

Reply:
they don't have targeting, they just spawn randomly headed for station

Reply:
Hmmm, looks like you're right. I could have sworn it targeted players. Maybe someone changed it already.